frontend/src/container/WorkspaceContent.jsx

```diff
@@ -110,7 +110,13 @@ class WorkspaceContent extends React.Component {
       else return
     } else wsToLoad = match.params.idws
 
-    this.loadContentList(wsToLoad)
+    this.loadContentList(wsToLoad).then(() => {
```

Comment:
don't use promise callback (`.then()`) prefer the usage of async function

Reply:
Done